Scenarios/Reachability/iOSReachability/BaseTableView.h

```diff
@@ -0,0 +1,24 @@
+//
+//  BaseTableView.h
+//  ObjCTableViewNoNib
+//
+//  Created by Luis Meddrano-Zaldivar on 4/21/17.
+//  Copyright © 2017 Luis Meddrano-Zaldivar. All rights reserved.
+//
```

Comment:
Each app needs the standard Microsoft MIT licensing notice. Check out the other samples to grab this.